docs/source/DeveloperDocumentation/MobileSdks/android-widevine-sdk.md

```diff
@@ -2,7 +2,7 @@
 
 # Android Widevine SDK
 
-VUDRMWidevine is an Android Archive (AAR) which can be used during the media rendering pipeline to provide a DRM plugin to ExoPlayer 2.12.2 which will work with Vualto DRM workflow. VUDRMWidevine has been developed to specifically manage the session DRM, allowing complete asset and player management.
+VUDRMWidevine is an Android Archive (AAR) which can be used during the media rendering pipeline to provide a DRM plugin to ExoPlayer 2.12.2 which will work with Studio DRM workflow. VUDRMWidevine has been developed to specifically manage the session DRM, allowing complete asset and player management.
```

Comment:
@macmagician @PerritonBranch I assume this will need to change again once the rebranding of the SDKs is complete?

Reply:
Yep once @macmagician has rebranded and released the new stuff this will need renaming. It may just benefit from a more complete overhaul but how needed that is will be clearer one Adam is finished.